### ISSUE TITLE
Add Engine support for playlists

### ISSUE THREAD

**Issue by mr-smidge** [xsco/libdjinterop#35]
Add support for playlists within the Engine Library format.

The format is nothing complicated, and is part of the `List` common structure that applies to playlists, history lists, prepare lists, and crates.

**Comment by audministrator** [comment]
Hopefully @mr-smidge can answer the question how far the develop is so far ?

**Comment by mr-smidge** [comment]
> Hopefully @mr-smidge can answer the question how far the develop is so far ?

I haven't started work on this feature yet (working on support for Engine 4.1.0/4.2.0/4.2.1), but I'm happy to make this the next priority.  There's been a few occasions when I've wanted to have this functionality too, so I'm motivated to do it :smile:.

**Comment by audministrator** [comment]
Really happy to hear that you keep this alive ! 
This is such a nice add-on for us, saves us a ton of work to move everything around ;-) 

The beautiful part of it that if everything is working properly, and as Engine DJ OS supports Dropbox as a data source.
We can just copy and past the database and tracks to 1 cloud location and sync to use it on multiple devices.

Which DJ software can tell that is supports cloud storage !!

**Comment by audministrator** [comment]
@mr-smidge  

Hi Again I saw that the  Engine 4.1.0/4.2.0/4.2.1 was merged and planned for v2.5.1 :+1:
Thanks a lot so much looking forward to this !

Can I ask if adding the playlist export is on the way as well ?
It would be so much more convenient as you can imagine.

**Comment by mr-smidge** [comment]
> Can I ask if adding the playlist export is on the way as well ? It would be so much more convenient as you can imagine.

Yes, it's next on my list :smile:!

**Comment by audministrator** [comment]
OK perfect ! 😄

Also just saw that ronso pushed my PR for the label change to be reviewed and incorporated in the next release (I hope)

[https://github.com/mixxxdj/mixxx/pull/14248#issuecomment-2639762086](url)

So everything comes together nicely !!

**Comment by mr-smidge** [comment]
> Wow ! really looking forward to this 😄 Thanks again for adding this very usefull feature 👍

Yes, I will start to integrate this into Mixxx as my next piece of work...!

**Comment by audministrator** [comment]
I was looking for the PR already, but could not find it yet 😄
So it probably want make the next release v  2.5.1  I guess ?

**Comment by audministrator** [comment]
Hi mr_smidge, 

I just tested the export to engine OS integration in the Mixxx beta v155 and all works well on Engine OS 4.3 !!!
Thanks for this great add-on that I will enjoy a lot !

I saw that the export playlists has the status CLOSED now in your repo 😄 
Does it mean it is ready and done as well ?
